### ISSUE TITLE
Wrong description in contributor checklist

### ISSUE THREAD

**Issue by karaxuna** [balena-io-modules/rendition#1398]
Is:

```
- [ ] I have regenerated screenshots for any affected components with `npm run generate-screenshots`
```

Should generate snapshots instead, right?

**Comment by LucianBuzzo** [comment]
@karaxuna Yep, I think this is a hangover from when we would do screenshot generation as well. Now we can actually rely entirely on the snapshotting functionality for regression testing.

**Comment by myarmolinsky** [comment]
Closed by #1462
